democracy/views/utils.py

```diff
@@ -105,9 +105,6 @@ def to_representation(self, images):
         else:
             images = images.public()
 
-        # Remove duplicated rows
```

Comment:
Was this addition ever necessary? It seems this has been added *after* the `ordering` field was defined for the `SectionImage` model, so surely it still does something? Apparently the reason was "Getting images throug the related name leads to return several time the same images, one per translation (due to the join). This commit orders the images by pk which remove the duplication."

So should we still have `.order_by('ordering')` or just `.distinct()` here?